Tests/DangerTests/GitDiffTests.swift

```diff
@@ -85,7 +85,7 @@ final class GitDiffTests: XCTestCase {
                                    FileDiff.Line(text: "struct ShellOutExecutor: ShellOutExecuting {", changeType: .unchanged),
                                ]),
                            ]),
```

Comment:
<!--
  0 failure: 
  1 warning:  Collection litera...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitDiffTests.swift;
  Line: 87;
-->

- :warning: Collection literals should not have trailing commas. (`trailing_comma`)